terraform/environments/preview/s3_buckets.tf

```diff
@@ -60,8 +60,12 @@ data "aws_iam_policy_document" "reports_bucket_policy_document" {
     effect = "Allow"
 
     principals {
-      identifiers = ["arn:aws:iam::${var.aws_dev_account_id}:role/developers"]
-      type        = "AWS"
+      identifiers = [
+        "arn:aws:iam::${var.aws_dev_account_id}:role/developers",
+        "arn:aws:iam::${var.aws_main_account_id}:role/jenkins-ci-IAMRole-1FIPDG9DE2CWJ",
```

Comment:
Another annoying question; why are two roles needed?

Reply:
The first one is the dev group, i.e. us - so I can access/debug the bucket in the console.
The second one is the Jenkins role - the script needs access to upload the file. We can probably remove this once the job is fully tested/run in prod.

Reply:
Right, thanks